servicebroker/test/routes/binding_test.js

```diff
@@ -76,18 +76,79 @@ describe('binding RESTful API', function() {
 
 
   context('Bind service ', function() {
```

Comment:
The happy path should be first. It is good to know that we test the basic function before we cover the failures.